docs/source/quickstart.mdx

```diff
@@ -11,7 +11,7 @@ Hello! This tutorial walks you through installing the Apollo Router and running
 
 If you have a bash compatible terminal you can download the latest version of the Apollo Router directly to your current directory.
 ```bash
-bash <(curl -fsSL https://raw.githubusercontent.com/apollographql/router/main/scripts/install.sh)  
+https://router.apollo.dev/download/nix/latest | bash
```

Comment:
@bnjjj If you can make the script POSIX, we can accept this suggestion to this PR before landing it and closing #962. 😄 

```suggestion
https://router.apollo.dev/download/nix/latest | sh
```